### PR TITLE
Set up ci/cd GitHub actions #8

### PR DESCRIPTION
**Pull request by Motouom**
Implenmented the CI/CD  workflow for the frontend

### CODE REVIEW

**Review by nancymuyeh — APPROVED** [review]
Great work there @Motouom

**Review by mbunwe-victor — APPROVED** [review]
Good job @Motouom

File: .github/workflows/ci-cd-backend.yml [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+on:
+  push:
+    branches:
+      - '**' 
+    tags-ignore: 
+      - 'v*'
+    paths:
+      - 'packit-backend/**'
+      - '.github/workflows/ci-cd-backend.yml'
```
Comment (Arielpetit) [inline .github/workflows/ci-cd-backend.yml:11]: Please can you add the checks for the PR?

**Review by Arielpetit — CHANGES_REQUESTED** [review]

File: .github/workflows/ci-cd-backend.yml (hunk 1 above)
Comment (Motouom) [reply .github/workflows/ci-cd-backend.yml]: done already

**Review by Arielpetit — APPROVED** [review]
Good

**Pull request merged**